docs/eventing/sinks/integration-sink/aws_sns.md

```diff
@@ -0,0 +1,35 @@
+# AWS Simple Notification Service Sink
+
+The `IntegrationSink` supports the Amazon Web Services (AWS) Simple Notification Service (SNS) service, through its `aws.sns` property.
+
+## Amazon credentials
+
+For connecting to AWS the `IntegrationSink` uses Kubernetes `Secret`, present in the namespace of the reSink. The `Secret` can be created like:
```

Comment:
will we document the permissions to give to those access keys?

Reply:
That's perhaps a good idea. I will see how camel handles this, on their docs, and borrow.